packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/TranscribedLines/TranscribedLinesContainer.js

```diff
@@ -38,6 +40,7 @@ function TranscribedLinesContainer (props) {
     return (
       <TranscribedLines
         lines={visibleLinesPerFrame}
+        marks={transcriptionTask.marks}
```

Comment:
Non-blocking: this is a bit redundant. We're already passing the task down as a prop and marks can be destructured and used in the child component already.

Reply:
Yeah. Without this, I found that changes to the `marks` array aren't picked up by the observer. I'm not sure why.

Reply:
Oh yeah, the mobx observable doesn't deep observe: https://mobx.js.org/intercept-and-observe.html#intercept--observe-

Reply:
I think we get caught out on this often forgetting this part of the observer function... maybe a small comment would be helpful here to note that?

Reply:
Sure. It did feel a bit hacky when I wrote this, but then I wrote the whole thing very quickly just to see if it would work.

Reply:
I think this is why they encourage connecting the store whenever it's needed and so if this does get refactored into two smaller separate components, the connector for the transcribed line could destructor the task ahead of time and pass in everything that's needed as props rather than the task object.

Reply:
I've destructured the `marks` prop in the `useStores` hook and added a comment.